### DISCUSSION TITLE
resurrect windows 7 style network flyout

### DISCUSSION THREAD

**Discussion by pyrates999** [valinet/ExplorerPatcher#1237] [Ideas]
Can you add an option that when clicking on the network system tray icon, it has the option to open the windows 7 style network flyout?

Currently explorer patcher allows for windows 8 and windows 10 style flyouts.

**Comment by Ben2557** [comment]
I'm also looking for it but the problem is related to Windows integration.

It would need to develop a new software displaying a menu with a network icon on the taskbar, as BatteryMode does for battery icon.

**Comment by pyrates999** [comment]
So the code to do this no longer exists in windows 10 or later?  Even after symbols have been downloaded?

**Reply by Ben2557** [reply]
Sadly I think so. Winaero Tweaker allows us to do the same thing and we only have the 10 flyout and the 8-like flyout only.
Since it's been a long time this software is being developed (developers aren't magicians), I believe this is related to Windows because Microsoft keep or remove things over time.
